shell/pages/c/_cluster/explorer/ConfigBadge.vue

```diff
@@ -16,26 +16,31 @@ export default {
     }
   },
   methods: {
-    customBadgeDialog() {
+    customBadgeDialog(ev) {
       this.$store.dispatch('cluster/promptModal', { component: 'AddCustomBadgeDialog', componentProps: { mode: _EDIT } });
     },
   },
 };
 </script>
 
 <template>
-  <div class="config-badge">
+  <div
+    class="config-badge"
+  >
     <div>
-      <a
+      <button
         class="badge-install btn btn-sm role-secondary"
         data-testid="add-custom-cluster-badge"
+        role="button"
+        tabindex="0"
         @click="customBadgeDialog"
+        @keyup.space="customBadgeDialog"
```

Comment:
Does a button require explicitly listening for `space`? iirc `space`/`enter` should trigger a click event in buttons.

Reply:
fixed as I turned it into a button